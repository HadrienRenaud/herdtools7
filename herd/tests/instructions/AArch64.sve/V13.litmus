AArch64 V13
(* Unsigned add reduction to scalar *)
{
}
P0                  ;
 MOV Z0.S,#4        ;
 PTRUE P0.S,VL3     ;
 UADDV D1,P0,Z0.S   ;

forall 0:V1.4S={12,0,0,0}
