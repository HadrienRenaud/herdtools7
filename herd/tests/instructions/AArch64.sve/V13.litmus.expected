Test V13 Required
States 1
0:V1.4S={12,0,0,0};
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (0:V1.4S={12,0,0,0})
Observation V13 Always 1 0
Hash=5e391bba5dda1e4a4f1f0a45c2c48975

